src/components/MetricsPane/charts/AggregateFishBiomas.jsx

```diff
@@ -26,14 +26,28 @@ export const AggregateFishBiomass = () => {
 
   const plotlyDataConfiguration = [
     {
-      x: surveyFishbeltBiomassValues.map((value) => (value > 5000 ? 5000 : value)),
+      x: surveyFishbeltBiomassValues.filter((value) => value <= 5000),
       type: 'histogram',
+      name: '',
       marker: {
         color: chartTheme.aggregateCharts.default.marker.color,
       },
-      xbins: { start: 0, size: 100 },
+      xbins: { start: 0, end: 5000, size: 100 },
+      hovertemplate: '%{x} kg/ha<br>%{y} surveys',
+      showlegend: false,
     },
-  ]
+    {
+      x: surveyFishbeltBiomassValues.filter((value) => value > 5000).map(() => 5000),
```

Comment:
I dont know if this is my tired brain, or if line 40 means all value will be mapped to `5000`. !s this `map` necessary?
If its some sort of hack, maybe make it a named `const` that explains why a bunch of 5000’s are needed?

Reply:
This is a workaround to group all 5000+ values to be one 5000 values, as requested by the Mermaid team, who produces charts from R,to prevent the chart from skewing too far to the left.

Might not worth to make it `const` since the array.filter is explaining itselft